libs/ledgerjs/script/crypto-assets-importer/importers/trc10/index.ts

```diff
@@ -79,6 +79,7 @@ const convertTRC10 = ({
   precision || 0,
   delisted,
   ledgerSignature,
+  true,
```

Comment:
We remove enableCounterValues in convertTRONTokens method, either we remove it fully or keep these method input